DSESwift/DataPoint.swift

```diff
@@ -63,48 +63,48 @@ public class DataPoint{
      * Set value of the data point.
      * @param value: value of the datapoint in String.
      **/
-    func setValue(value: String) {
+    public func setValueWithString(value: String) {
         self.value = value
     }
     
     /**
      * Return the timestamp in Int with milliseconds accuracy
      **/
-    func getTimeInMillis() -> Int {
-        return Int(self.time.timeIntervalSince1970 * 1000)
+    public func getTimeInMillis() -> Double {
+        return round(self.time.timeIntervalSince1970 * 1000)
     }
     
     /**
      * Return the value in Int.
      **/
-    func getValueInInt() -> Int {
+    public func getValueInInt() -> Int {
```

Comment:
DataPoint class itself does not know about its value's datatype, and values are stored as string, so someone has to provide the datatype to cast into either by parameter or the function name. What do you suggest?